Gemfile.lock

```diff
@@ -0,0 +1,263 @@
+PATH
```

Comment:
Generally we don't store a lock file. Could you drop this?

Reply:
Done.